sheetsAuth.js

```diff
@@ -83,6 +83,28 @@ function formatSheetResults(auth) {
   })
 }
 
+/* for getting datas from zoom raw data output sheet */
+function getAttendanceDatas(auth) {
+  return new Promise( (resolve, reject) => {
```

Comment:
We should consider standardizing either on Promises or Async/Await for a future iteration to be consistent.  I'm partial to the latter, you use it here as well and Michelle is also using it.